fdbserver/SimKmsConnector.actor.cpp

```diff
@@ -221,6 +222,20 @@ ACTOR Future<Void> blobMetadataLookup(KmsConnectorInterface interf, KmsConnBlobM
 
 	wait(delay(deterministicRandom()->random01())); // simulate network delay
 
+	// buggify errors or omitted tenants in response
+	if (!g_simulator->speedUpSimulation && BUGGIFY_WITH_PROB(0.01)) {
```

Comment:
nit: do we need to check `g_network->isSimulated()` ; otherwise it might crash for `non simulation` tests? Not sure if there are any such tests. `g_simultor*` is `null` for such tests.

Reply:
I didn't check since it's SimKmsConnector, but we could.
I believe this runs as the perf kms connector too but you'd also have to have buggify on in that case.

Reply:
Yeah `PerfKmsConnector` is meant to be run in `non-simulation` mode; hence, my comment. So far we have not `g_simulator` object in SimKms, it would good to guard the access to avoid SEGVs if any. 
Thanks

Reply:
Sounds good, will do a follow-up PR